util/tidy.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2013 The Servo Project Developers.
```

Comment:
Output of `git shortlog -sn python/tidy.py` in the Servo repository is below. @bors-servo is a bot that only makes merge commits. The original file does lots more stuff, so probably not everyone listed has touched the parts kept in this PR. But if copyright/licensing is a problem, maybe it’d be easier to rewrite this script from scratch?

```
    21  Corey Farwell
    17  bors-servo
    11  Ms2ger
     5  Josh Matthews
     4  Simon Sapin
     3  Bogdan Cuza
     3  Brandon Fairchild
     3  Daan Sprenkels
     3  Dhananjay Nakrani
     3  Duncan Keall
     3  Manish Goregaokar
     3  Michael Howell
     3  Ravi Shankar
     3  wilmoz
     2  Anthony Ramine
     2  Bobby Holley
     2  Gilles Leblanc
     2  Guillaume Gomez
     2  Jack Moffitt
     2  João Oliveira
     2  Matt Brubeck
     2  PythonNut
     2  jmr0
     1  Attila Dusnoki
     1  Duncan
     1  Greg Guthe
     1  Hugo Thiessard
     1  Jaydeep
     1  Johann Tuffe
     1  Martin Robinson
     1  Mathieu Rheaume
     1  Mike Fairhurst
     1  OneKorg
     1  Paul Padier
     1  Pierre Chevalier
     1  Prabhjyot Singh Sodhi
     1  Rahul Sharma
     1  Simon Martin
     1  Till Schneidereit
     1  UK992
     1  erneyja